Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs

```diff
@@ -57,8 +57,8 @@ public override void Update(float frameTime)
         /// </summary>
         private void ResetTimer(BasicStationEventSchedulerComponent component)
         {
-            // 5 - 25 minutes. TG does 3-10 but that's pretty frequent
-            component.TimeUntilNextEvent = _random.Next(300, 1500);
+            // 15-35 minutes
```

Comment:
15-35 seems like it may be a little high. I'll leave it to other opinions.

Reply:
That's how it was previously